lib/rails-jquery-autocomplete/orm/active_record.rb

```diff
@@ -24,27 +24,39 @@ def active_record_get_autocomplete_items(parameters)
 
         items = (::Rails::VERSION::MAJOR * 10 + ::Rails::VERSION::MINOR) >= 40 ? model.where(nil) : model.scoped
 
-        scopes.each { |scope| items = items.send(scope) } unless scopes.empty?
-
         items = items.select(get_autocomplete_select_clause(model, method, options)) unless options[:full_model]
         items = items.where(get_autocomplete_where_clause(model, term, method, options)).
             limit(limit).order(order)
         items = items.where(where) unless where.blank?
 
+        if options[:unique]
+          scopes <<  lambda { unscope(:select).select("MIN(#{table_name}.#{model.primary_key}) as #{model.primary_key}, #{table_name}.#{method}").group(method) }
```

Comment:
Line is too long. [161/80]<br>Unnecessary spacing detected.<br>Use the new lambda literal syntax `-> {...}`.<br>Operator `<<` should be surrounded by a single space.